monitoring/sysdig/docs/deployment.md

```diff
@@ -14,8 +14,8 @@ oc new-project devops-sysdig --description='BC Gov DevOps Platform Sysdig Monito
 oc create serviceaccount sysdig-agent
 oc adm policy add-scc-to-user privileged -n devops-sysdig -z sysdig-agent
 oc adm policy add-cluster-role-to-user cluster-reader -n devops-sysdig -z sysdig-agent
-oc create secret generic sysdig-agent --from-literal=access-key=<your sysdig access key> -n devops-sysdig
```

Comment:
We need to keep `oc create secret generic sysdig-agent`

Reply:
Added back `oc create secret generic sysdig-agent` line.